src/main/app/case/case.ts

```diff
@@ -250,6 +265,9 @@ export interface Case {
   jurisdictionAgree?: YesOrNo;
   reasonCourtsOfEnglandAndWalesHaveNoJurisdiction?: string;
   inWhichCountryIsYourLifeMainlyBased?: string;
+  isFathersNameOnCertificate?: string;
+  isFatherStillAlive?: string;
+  isFathersAddressKnown?: string;
```

Comment:
are these variables required now?

Reply:
What is the use of isFatherStillAlive as already I can see birthFatherStillAlive variable?

Reply:
Again birthFatherAddressKnown variable is already there then what is the use of isFathersAddressKnown?